storage_json.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	model "github.com/golang-bristol/beer-model"
+	"github.com/nanobox-io/golang-scribble"
+)
+
+// StorageJSON is the data storage type using JSON file
+type StorageJSON struct {
+	db *scribble.Driver
+}
+
+var (
+	collectionBeer   = "beers"
+	collectionReview = "reviews"
+)
+
+func newStorageJSON(location string) (*StorageJSON, error) {
+	var err error
+
+	stg := new(StorageJSON)
+
+	stg.db, err = scribble.New(location, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return stg, nil
+}
+
+// SaveBeer insert or update beers
+func (s *StorageJSON) SaveBeer(beers ...model.Beer) error {
+	for _, beer := range beers {
+		var err error
+		var resource = fmt.Sprintf("%d", beer.ID)
+
+		beersFound, err := s.FindBeer(model.Beer{ID: beer.ID})
+		if err != nil {
+			return err
+		}
+
+		if len(beersFound) > 0 {
+			err = s.db.Delete(collectionBeer, resource)
+			if err != nil {
+				return err
+			}
+		}
+
+		err = s.db.Write(collectionBeer, resource, beer)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// SaveReview insert or update reviews
+func (s *StorageJSON) SaveReview(reviews ...model.Review) error {
+	for _, review := range reviews {
+		var err error
+		var resource = fmt.Sprintf("%d", review.ID)
```

Comment:
Same answer about collections and resources names used by `golang-scribble`, but I agree we could improve the way we assign IDs to beers and reviews.

Reply:
Yep, using the ID as the identifier is totally fine by me. I was just thinking of the way we assign the ID - I think it would be better to get the next available one rather than take it from the input, but I suppose that could be an improvement for later.